README.md

```diff
@@ -1 +1,288 @@
-# amar-hello-microverse
\ No newline at end of file
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<!-- <div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <!-- <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/> -->
+
+  <!-- <h3><b>Hello Microverse</b></h3>
+
+</div> --> 
+
+<!-- TABLE OF CONTENTS -->
+
+<!-- # 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license) -->
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Hello Microverse <a name="about-project"></a>
+
+<!-- > Describe your project in 1 or 2 sentences. -->
+
+**Hello Microverse** is a...
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<!-- > Describe the tech stack and include only the relevant sections that apply to your project. -->
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>HTML and CSS</li>
+  </ul>
+</details>
+
+<!-- <details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">Express.js</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details> -->
+
+<!-- Features -->
+
+<!-- ### Key Features <a name="key-features"></a>
+
```

Comment:
- [ ] It appears as though you commented out a major part of your README. Please note that all sections of the README are mandatory except the FAQ. Currently, the image below shows how your README is and that is not professional at all. 

![image](https://user-images.githubusercontent.com/84156714/234725678-5a36c6b4-3aac-4da2-9e54-bf5beec63e0d.png)

- [ ] Please so the following to your README file.
   - All sections of this README template are mandatory except the FAQ which is optional.
   - Add a detailed project description. 
   - For the Key Features section, kindly highlight what this project is all about. This will make it easy for those who go through this repository to know what was implemented by reading the README.md file. 👍
   - Since this project does not require any demo, you can leave out the Live Demo section.
   - For the Getting Started Section, and the Prerequisite Section, kindly add tools that are needed before you can clone this repository on a local machine. Tools like:
        - Installation of Git
        - Installation of NPM
        - A working browser
        - Internet connection
        - A code editor like Visual Studio Code
        - Installation of Live Server plugging in Visual Studio Code
        - Any other tool you can think about
   - For the Setup section, kindly add instructions for cloning your repository on a local machine 👍
   - For the Install section, kindly add instructions for setting up linters used in this project
   - For the Usage section, include instructions for opening this project on a browser. For example, opening the project by launching the Live Server plugging in visual studio code or any other means that you know.
   - For future features, add at least things you will do in the future to improve this project.